packages/react-components/components/PhoneNumberInput.tsx

```diff
@@ -3,12 +3,17 @@ import ValidatedTextInput, {
   PhoneValidatorProps,
 } from '@celo/react-components/components/ValidatedTextInput'
 import colors from '@celo/react-components/styles/colors'
+import SmsRetriever from '@celo/react-native-sms-retriever'
```

Comment:
If you're going to add a dependency to this here in react-components, you should modify its package.json